src/main/java/net/javadiscord/javabot/data/config/guild/ModerationConfig.java

```diff
@@ -96,4 +98,6 @@ public Role getAdminRole() {
 	public Role getExpertRole() {
 		return this.getGuild().getRoleById(this.expertRoleId);
 	}
+
+	public long getMinActiveHoursToSendMedia() { return this.minActiveHoursToSendMedia; }
```

Comment:
Remove this, the getters for Roles/Channels are there to return them as an Role or Channel object instead of the id